src/main/java/io/github/redstoneparadox/oaktree/client/gui/control/EntityPreviewControl.java

```diff
@@ -0,0 +1,32 @@
+package io.github.redstoneparadox.oaktree.client.gui.control;
+
+import io.github.redstoneparadox.oaktree.client.gui.ControlGui;
+import net.minecraft.client.gui.screen.ingame.InventoryScreen;
+import net.minecraft.client.util.math.MatrixStack;
+import net.minecraft.entity.LivingEntity;
+
+public class EntityPreviewControl extends Control<EntityPreviewControl> {
```

Comment:
Class is missing Javadoc (Yes, I know all of mine don't have it but I'm lazy) and I'm thinking there should be an option to make the entity look at the mouse; I'm fine with that being limited to the player.

Reply:
Is it good now?